### PR TITLE
feat: add help-now-section

### PR DESCRIPTION
**Pull request by OlhaPo**
This PR represents adding help-now section with markup (1440px and larger screens) according to figma design (approved page).

Responsive design will be added in a future update.

### **Commit updates** 
c8a51a6: modify the IButton interface and add functionality to correctly switch between currencies while ensuring that the appropriate sum button is activated based on the selected currency. In future commits default currency selection logic will be impemented based on selected language (for EN the default currency will be set to Euro, PL - Zloty, UA - UAH)

### CODE REVIEW

File: messages/en.json [hunk 1]
```diff
@@ -35,6 +35,19 @@
           "imageAlt": "lottery"
         }
       }
+    },
+    "paymentSection": {
```
Comment (baza-trainee) [inline messages/en.json:39]: rename: HelpNowSection. as component and folder name

Comment (biryukov-m) [inline messages/en.json:39]: rename: HelpNowSection. as component and folder name

File: messages/pl.json [hunk 2]
```diff
@@ -35,6 +35,19 @@
           "imageAlt": "loteria"
         }
       }
+    },
+    "paymentSection": {
```
Comment (biryukov-m) [inline messages/pl.json:39]: rename: HelpNowSection. as component and folder name

File: messages/ua.json [hunk 3]
```diff
@@ -35,6 +35,19 @@
           "imageAlt": "лотерея"
         }
       }
+    },
+    "paymentSection": {
```
Comment (biryukov-m) [inline messages/ua.json:39]: rename: HelpNowSection. as component and folder name

File: src/features/home/sections/HelpNowFormSection/HelpNowFormSection.module.scss
Comment (biryukov-m) [inline src/features/home/sections/HelpNowFormSection/HelpNowFormSection.module.scss:1]: rename this file from HelpNowFormSection to HelpNowSection
and rename folder too

File: src/features/home/sections/HelpNowFormSection/index.tsx [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+
+const DEFAULT_CURRENCY = 'uah';
+const DEFAULT_TYPE = 'Одноразовий платіж';
+const DEFAULT_SUM_UAH = '20';
+const DEFAULT_SUM_EUR = '5';
+const DEFAULT_SUM_ZLOTY = '10';
+
+const HelpNowSection = () => {
+  const [selectedCurrency, setSelectedCurrency] = useState(DEFAULT_CURRENCY);
+  const [selectedType, setSelectedType] = useState(DEFAULT_TYPE);
```
Comment (biryukov-m) [inline src/features/home/sections/HelpNowFormSection/index.tsx:16]: typescript, for ex.
`useState<CURRENCY>`
`useState<PAYMENT_SUBSCTIPTION>`

File: src/features/home/sections/HelpNowFormSection/index.tsx [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+      size: 'medium',
+      value: 'інша сума',
+    },
+  ];
+
+  const DONATE_TYPE: IPayment[] = [
+    {
+      variant: 'pay',
+      size: 'medium',
+      value: 'Щомісячна підписка',
```
Comment (biryukov-m) [inline src/features/home/sections/HelpNowFormSection/index.tsx:82]: useTranslation

File: src/features/home/sections/HelpNowFormSection/index.tsx [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+    },
+    {
+      variant: 'pay',
+      size: 'medium',
+      value: 'Одноразовий платіж',
```
Comment (biryukov-m) [inline src/features/home/sections/HelpNowFormSection/index.tsx:87]: useTranslation

**Review by biryukov-m — CHANGES_REQUESTED** [review]
Change code from requested in comments + 

In helpnow component: 
move non-rendering logic to custom hook in features\home\hooks\use....  
Separate all rendering logic from data logic:
`CURRENCY_VALUES DONATE_SUM DONATE_TYPE handleCurrencyChange handleSumChange handleTypeChange  formatCurrencyLabel ` - move to hook 
`formatCurrencyLabel` move to helpers _features\home\utils\_


And take a look at src/hooks/usePayment hook - example from Ihor.
You can make in two steps: make your own hook, then connect with logic from Ihor's usePayment. Or make in one

File: src/features/home/sections/HelpNowFormSection/index.tsx [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+  };
+
+  const formatCurrencyLabel = (value: string) => {
+    if (value === 'інша сума') return value;
+    switch (selectedCurrency) {
+      case 'eur':
+        return `${value} euro`;
+      case 'zloty':
+        return `${value} zł`;
+      default:
```
[the remark this answers is not in the document]
Comment (OlhaPo) [reply src/features/home/sections/HelpNowFormSection/index.tsx:117]: Did you mean to do uah in its onw case? Ignore for now that value: string, I'll change it using type

 const formatCurrencyLabel = (value: string) => {
    if (value === 'інша сума') return value;
    switch (selectedCurrency) {
      case 'eur':
        return `${value} euro`;
      case 'zloty':
        return `${value} zł`;
      case 'uah':
        return `${value} грн`;
      default:
        return `${value}`;
    }
  };

[the remark this answers is not in the document]
Comment (biryukov-m) [reply src/features/home/sections/HelpNowFormSection/index.tsx]: yes

File: src/features/home/sections/HelpNowFormSection/index.tsx [hunk 8]
```diff
@@ -0,0 +1,179 @@
...
+      setSelectedSum(DEFAULT_SUM_EUR);
+    } else if (value === 'zloty') {
+      setSelectedSum(DEFAULT_SUM_ZLOTY);
+    } else {
+      setSelectedSum(DEFAULT_SUM_UAH);
+    }
+  };
+
+  const handleSumChange = (value: string) => {
+    setSelectedSum(value);
```
[the remark this answers is not in the document]
Comment (OlhaPo) [reply src/features/home/sections/HelpNowFormSection/index.tsx]: still need to be done

**Review by biryukov-m — APPROVED** [review]

**Review by baza-trainee — APPROVED** [review]

**Pull request merged**